include/seqan3/core/simd/view_to_simd.hpp

```diff
@@ -211,7 +211,7 @@ class view_to_simd<urng_t, simd_t>::iterator_type
      */
     using reference = std::span<std::ranges::range_value_t<chunk_type>>; //!< The reference type.
     using value_type = chunk_type; //!< The value type.
-    using pointer = chunk_type *; //!< The pointer type.
+    using pointer = void; //!< The pointer type.
```

Comment:
This has no implications? I didn't know that setting the pointer type as void is valid :)

Reply:
Yes. The pointer type is only required if you can use the operator->. Since, we return a proxy we can only make this safe by returning a unique_ptr. But the question is if that is something you want to have. Right now it merely means that it cannot be a CPP17Iterator. But I can live with that.